image.ml

```diff
@@ -68,9 +68,11 @@ exception Corrupted_image of string
 exception Not_yet_implemented of string
 
 let create_rgb ?(alpha=false) ?(max_val=255) width height =
-  assert (1 <= max_val && max_val <= 65535);
-  assert (width > 0);
-  assert (height > 0);
+  if not (1 <= max_val && max_val <= 65535) then
```

Comment:
I think it would be better to use `invalid_arg` here, and in fact mostly everywhere in this file.

Reply:
That makes sense, will change them. :)